### ISSUE TITLE
feature request: set values below a threshold as NA

### ISSUE THREAD

**Issue by cvanderaa** [rformassspectrometry/QFeatures#135]
The normalization function in [`SCeptre`](https://github.com/bfurtwa/SCeptre/blob/29a1e5a778096e2c40261c50335471159569ac5f/sceptre/sceptre.py#L625-L631) (from Schoof et al. 2021 paper) performs at some moment an NA thresholding, that is quantitative values below a given threshold are set as `NA`. I personally think this should not be performed in a normalization function, but it could be implemented in a separate function. This would be very similar to `zeroIsNA` but it must require an additional argument that provides the threshold below which the data is considered missing. 

Do you think this functionality is worth being included in `QFeatures`? Or do we already have this but I missed it?

**Comment by lgatto** [comment]
I don't think we should change `zeroIsNA(x)` to  `zeroIsNA(x, ..., threshold = 0)`, but we could have a function `setToZero(x, min)` that sets values <= min to 0. Or could you not simply create that function and apply it on the fly (as a example of how to process assays with user-provided functions) ?
